modules/_labs/menu/react/stories/stories.tsx

```diff
@@ -33,7 +33,7 @@ const createMenuItems = (hasIcons?: boolean): StoryMenuItemProps[] => {
       icon: hasIcons ? uploadCloudIcon : undefined,
     },
     {
-      text: `Second Item with really really really long label`,
+      text: `Second Item (with a really really really long label)`,
```

Comment:
Parentheses were added to keep visual consistency with the other labels, and the "a" was added as a grammar check. 😄